### PR TITLE
React to VerdictReportComputedTask

### LINKED ISSUE #2231: [Concent] UC1 / Force Report - react to `VerdictForceReportComputeTask`
as a Requestor
* react to `VerdictReportComputedTask`

### PR DESCRIPTION
**Pull request by jiivan**
fixes: #2231

### CODE REVIEW

File: golem/network/concent/received_handler.py [hunk 1]
```diff
@@ -40,13 +40,49 @@ def on_concents_service_refused(self, msg):
...
-    @handler_for(message.concents.ForceReportComputedTask)
-    def on_concents_force_report_computed_task(self, msg):
-        concent_helpers.process_report_computed_task(
-            msg=msg.report_computed_task,
-            task_session=None,
+    @handler_for(message.concents.VerdictReportComputedTask)
+    def on_verdict_report_computed_task(self, msg):
+        """Verdict is forced by Concent on Requestor
+
+        Requestor should as it has sent AckReportComputedTask by himself.
```
Comment (shadeofblue) [inline golem/network/concent/received_handler.py]: the sentence lacks a verb... I'd suggest `act` ? and for proper grammar -> `should act as if it _had_ sent` #nitpicking #sorry ;)

Comment (jiivan) [reply golem/network/concent/received_handler.py]: Did you mean "_this_ sentence"? :P

File: golem/network/concent/received_handler.py [hunk 2]
```diff
@@ -40,13 +40,49 @@ def on_concents_service_refused(self, msg):
...
+        )
+        for ttc in ttcs_tuple:
+            try:
+                self.task_server.keys_auth.ecc.verify(
+                    sig=ttc.sig,
+                    inputb=ttc.get_short_hash(),
+                )
+            except msg_exceptions.InvalidSignature:
+                logger.error(
+                    '[CONCENT] Received fake TaskToCompute from Concent: %s',
```
Comment (shadeofblue) [inline golem/network/concent/received_handler.py:65]: :D naughty Concent! naughty! ;)

**Review by shadeofblue — APPROVED** [review]
looks good

**Pull request merged**